modules/friendlytag.js

```diff
@@ -928,7 +928,6 @@ Twinkle.tag.redirectList = {
 			{ tag: 'R from alternative name', description: 'redirect from a title that is another name, a pseudonym, a nickname, or a synonym' },
 			{ tag: 'R from ambiguous sort name', description: 'redirect from an ambiguous sort name to a page or list that disambiguates it' },
 			{ tag: 'R from former name', description: 'redirect from a former name or working title' },
```

Comment:
```suggestion
			{ tag: 'R from former name', description: 'redirect from a former or historic name or a working title' },
```
Might as well change the description of {{R from former name}} to so that search hits for "historic" will reach here instead.